app/views/oauth2/authorize.js

```diff
@@ -86,4 +86,7 @@ export default ['$scope', '$http', '$browser', '$window', '$location', 'user', f
     } // if(rOwnerAuthenticated)
   }; // scope.authorize
 
+  $(document).ready(function() {
+    $scope.authorize();
+  });
```

Comment:
I think it nees to be in a apply stage:
```suggestion
  $(document).ready(function() {
    $scope.$applyAsync(function() {
        $scope.authorize();
    });
  });
```